.github/workflows/pipeline.yml

```
@@ -0,0 +1,51 @@
+name: CarBuddy CI/CD Pipeline
+
+on:
+    push:
+        branches:
+        - main
+    pull_request:
+        branches:
+        - main
+
+jobs:
+  build:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)